Source/main.cpp

```diff
@@ -181,7 +181,7 @@ void main_main ()
     GpuArray<Real,AMREX_SPACEDIM> dx = geom.CellSizeArray();
 
     // Nghost = number of ghost cells for each array
-    int Nghost = 1;
+    int Nghost = 2;
 
     // Ncomp = number of components for each array
     int Ncomp = 1;
```

Comment:
I would get rid of Ncomp and replace the one instance we use of Ncomp below with "1"

Reply:
Fixed